### ISSUE TITLE
User feedback: claiming of issues

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#403]
From a user

> 1. I don't like locking the issue to one person right away. I get why you do it, but it's tough on the issuer. I think a more middleground approach might be to allow people to start working on it and lock the best one after maybe 24 hours. In my case, it became clear the guy couldn't do it fairly quickly (my issue was deceptively advanced, which is my mistake for offering something so difficult), but it would have been nice to learn that before locking him in.

# Ideas

1. Maybe allow the submitter to select one of those who has started work to be the 'blessed' one(s) to carry out the bounty.
2. If the submitter does not 'bless' any particular submitter, then anyone may work on the issue.

**Comment by owocki** [comment]
cc @pixelantdesign @mbeylin in case ya'll have opinions on this

**Comment by PixelantDesign** [comment]
@owocki 
Not sure if you've seen this problem come up again with other funders and I know my response is way late here but...I think it would be unfair to ask a developer to start work and then see if a better developer comes along to lock in the best one.

I think what could help here are: 
- a well defined task from a funder perspective
- more accurate measure of developer levels (beginner, intermediate, advanced) how ever the engine might work.
- early implementations of a feedback mechanism so that funders and contributors can comment on the experience they've had working together. 

@vs77bb

**Comment by mkosowsk** [comment]
I agree with @PixelantDesign's points and think Gitcoin should tread relatively lightly here... don't think the community wants to go down the Spec work path that plagues so many other platforms.

An additional idea could be to have an optional step in the flow after "Open" but before "Work Started" with something like "Interest Expressed". When the first dev expresses interest in a given issue, it could kick off a 24 hour period where other devs could express interest if so inclined. Then the Funder could decide which dev to 🙏🏻  after going through their profiles and chatting a bit with each.

Pros to this Approach:

0. Issuer gets some selection in the choice of dev. Could be very nice for trickier issues that require some expertise.
1. Devs don't do work they won't get paid for.

Cons to this Approach:

0. "Interest Expressed" muddies up the very succinct flow UX flow that was just established

I think pros may outweigh the cons here but am definitely open to feedback... I tend to have too much crap going on in my personal projects, they usually end up looking something like this 😂

![incredible_machine](https://user-images.githubusercontent.com/7970539/37631411-02bf12b0-2bc0-11e8-9280-6b6c122fbe7c.png)

**Comment by owocki** [comment]
> I think it would be unfair to ask a developer to start work and then see if a better developer comes along to lock in the best one.

i agree.

**Comment by vs77bb** [comment]
:+1:
